src/netaddress.cpp

```diff
@@ -328,28 +482,71 @@ enum Network CNetAddr::GetNetwork() const
     return m_net;
 }
 
+static std::string IPv6ToString(Span<const uint8_t> a) {
+    assert(a.size() == ADDR_IPV6_SIZE);
+    // clang-format off
+    return strprintf("%x:%x:%x:%x:%x:%x:%x:%x",
+                     ReadBE16(&a[0]),
+                     ReadBE16(&a[2]),
+                     ReadBE16(&a[4]),
+                     ReadBE16(&a[6]),
+                     ReadBE16(&a[8]),
+                     ReadBE16(&a[10]),
+                     ReadBE16(&a[12]),
+                     ReadBE16(&a[14]));
+    // clang-format on
```

Comment:
8d2b60cb With clang-format-12 these two `clang-format` declarations don't seem needed for me; ignore me if they are needed for other versions.

Reply:
clang-format 12 puts everything in one line for me.